sdcm/utils/docker_utils.py

```diff
@@ -303,6 +303,16 @@ def get_container_port(cls, instance: object, name: str, port: Union[int, str])
         except (IndexError, KeyError, ):
             return None
 
+    @classmethod
+    def get_bind_path(cls, instance: object, name: str, container_dir: str) -> str:
```

Comment:
`get_host_volume_path`
get's the directory on the host that was mounted to container_dir

Reply:
Done